parm/params_grib2_tbl_new.text

```diff
@@ -751,6 +751,8 @@
    0  19    36   0 PSNOWS
 !  Added more parameters in 12/03/2019
    0  19    37   0 ICESEV
+!  Added more parameters in 11/01/2023
+   0  19    50   0 CITEDR
```

Comment:
@hsinmulin-NOAA There is no this variable in the table 4.2-0-19. You may send a request to @AndrewBenjamin-NOAA for the table update.

Reply:
@AndrewBenjamin-NOAA Thanks for updating g2tmpl. Can you also update the table at https://www.nco.ncep.noaa.gov/pmb/docs/grib2/grib2_doc/grib2_table4-2-0-19.shtml so RRFS developers can get reference of this new product?

Reply:
Yes, I am doing that currently for the v32 release that just came out. I've been out the last few days so I haven't had a chance to complete every table, but I have added CITEDR to the list of products that need to be added to Table 4.2-0-19.

Reply:
@AndrewBenjamin-NOAA When the updated params_grib2_tbl_new* files are available, please pass them to @hsinmulin-NOAA so he may include in this PR.